lbrynet/core/Wallet.py

```diff
@@ -1455,6 +1492,24 @@ def _save_wallet(self, val=None):
         self.wallet.storage.write()
         return defer.succeed(val)
 
+    def decrypt_wallet(self):
+        if not self.wallet.use_encryption:
+            return True
```

Comment:
not sure i like this. `decrypt_wallet` shouldn't say it decrypted something when it didnt